src/services/widgets/widgets.svc.coffee

```diff
@@ -74,22 +74,39 @@ angular
         setting.initialize()
       return true
 
-    @updateWidgetSettings = (widget, needContentReload=true) ->
+    @updateWidgetSettings = (widget, needContentReload=true, ignoreReach=false) ->
       widget.isEditMode = false
-
       if _.isEmpty(widget.settings)
         $log.warn("ImpacWidgetsSvc: Tried to update widget: #{widget.id} with no settings", widget)
         return false
 
+      changedGlobalSetting = _.find widget.settings, (setting)-> setting.reach == 'dashboard'
+
+      if (changedGlobalSetting && !ignoreReach)
+        return _self.updateAllSameWidgets(widget, ImpacDashboardsSvc.getCurrentDashboard(), changedGlobalSetting)
+
       widget.isLoading = true if needContentReload
       meta = _.reduce(_.map(widget.settings, (set) -> set.toMetadata() ), (result, setMeta) -> angular.merge(result, setMeta))
-      
-      return _self.update(widget, { metadata: meta }).then(
+
+      _self.update(widget, { metadata: meta }).then(
         (updatedWidget) ->
           if needContentReload
             _self.show(updatedWidget).finally( -> updatedWidget.isLoading = false )
       )
 
+    @updateAllSameWidgets = (widget, dashboard, settings) ->
```

Comment:
`WidgetsSvc.updateAllSameWidgets`:
  - this logic should be placed in `DashboardsSvc`, as it concerns the update of a dashboard
  - **important**: you cannot test the widget's name to identify the widgets that have to be changed: the widget's name can be changed by the user... Please stick to the logic described in previous comment: (apply `toMetadata()` on all the settings of all the widgets ; the widgets to be updated will be the ones containing at least one setting which has the same key as the setting passed in parameter)